cmd/cmd_instance.go

```diff
@@ -48,6 +48,8 @@ func instanceCreateCommand() *cobra.Command {
 	PersistCreateInstanceFlags(cmdInstanceCreate.PersistentFlags())
 	cmdInstanceCreate.PersistentFlags().StringP("instance-name", "i", "", "instance name (overrides default instance name format)")
 	cmdInstanceCreate.PersistentFlags().StringP("instance-group", "", "", "instance group")
+	cmdInstanceCreate.PersistentFlags().StringP("aws-ipn", "", "", "AWS instance profile name")
```

Comment:
if we have the equivalent of the GCP version of this we should use this and if not just make it agnostic so it's not aws specific